x/incentives/keeper/distribute.go

```diff
@@ -228,64 +228,37 @@ func (k Keeper) doDistributionSends(ctx sdk.Context, distrs *distributionInfo) e
 func (k Keeper) distributeSyntheticInternal(
 	ctx sdk.Context, gauge types.Gauge, locks []lockuptypes.PeriodLock, distrInfo *distributionInfo,
 ) (sdk.Coins, error) {
-	totalDistrCoins := sdk.NewCoins()
-	denom := gauge.DistributeTo.Denom
+	qualifiedLocks := k.lk.GetLocksLongerThanDurationDenom(ctx, gauge.DistributeTo.Denom, gauge.DistributeTo.Duration)
```

Comment:
Question: is this change not susceptible to the recent gas reordering bug because it happens in BeginBlock?

cc: @ValarDragon

Reply:
It was only getting merkelized due to it being in a txs flow before